ranch/utils.py

```diff
@@ -130,7 +130,17 @@ def ox_graph(nodes_df, links_df):
     graph_links["id"] = graph_links["shstReferenceId"]
     graph_links["key"] = graph_links["shstReferenceId"]
```

Comment:
For the new links and nodes in the network, they probably do not have `shst_node_id`, `shstReferenceId`, `u`, `v`. If so, we cannot use them as unique identifier any more. can we use model node id, link id, and a/b nodes instead?

Reply:
I see what you did. You assigned them unique shst and osm IDs before rerouting. this is fine for now.